backend/capellacollab/__main__.py

```diff
@@ -153,18 +131,31 @@ async def healthcheck():
 app.include_router(router, prefix="/api/v1")
 
 
+async def exception_handler(
+    request: fastapi.Request, exc: Exception
+) -> fastapi.Response:
+    if isinstance(exc, core_exceptions.BaseError):
+        return await exception_handlers.http_exception_handler(
+            request,
+            fastapi.HTTPException(
+                status_code=exc.status_code,
+                detail={
+                    "title": exc.title,
+                    "reason": exc.reason,
+                    "err_code": exc.err_code,
+                },
+            ),
+        )
+    else:
+        return await exception_handlers.http_exception_handler(
+            request,
+            fastapi.HTTPException(status_code=500, detail=str(exc)),
+        )
```

Comment:
This is not reachable since you only register the exception handler for subclasses of BaseError.